qubesadmin/app.py

```diff
@@ -316,7 +317,8 @@ def add_new_vm(self, cls, name, label, template=None, pool=None,
         return self.domains[name]
 
     def clone_vm(self, src_vm, new_name, new_cls=None, pool=None, pools=None,
-                 ignore_errors=False, ignore_volumes=None):
+                 ignore_errors=False, ignore_volumes=None, ignore_devices=True):
```

Comment:
Please make the default False. It is a change in behavior, but I think the previous one was a bug.